subprojects/maven/src/main/java/org/gradle/api/publication/maven/internal/BasePomFilterContainer.java

```diff
@@ -75,6 +76,20 @@ public MavenPom pom(String name, Closure configureClosure) {
         return ConfigureUtil.configure(configureClosure, pom(name));
     }
```

Comment:
It would be better if the `Closure` variants called through to the `Action` variants.